### ISSUE TITLE
用xDeepFM和DeepFM模型，结果几乎一样，甚至稍低

### ISSUE THREAD

**Issue by ArtificialZeng** [shenweichen/DeepCTR#482]
Use tf.where in 2.0, which has the same broadcast rule as np.where
Train on 6129770 samples, validate on 1532443 samples
2022-07-14 16:28:09.045936: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
Epoch 1/10
6129770/6129770 - 149s - loss: 0.1881 - binary_crossentropy: 0.1858 - val_loss: 0.1871 - val_binary_crossentropy: 0.1841
Epoch 2/10
6129770/6129770 - 157s - loss: 0.1860 - binary_crossentropy: 0.1828 - val_loss: 0.1863 - val_binary_crossentropy: 0.1832
Epoch 3/10
6129770/6129770 - 158s - loss: 0.1856 - binary_crossentropy: 0.1823 - val_loss: 0.1860 - val_binary_crossentropy: 0.1829
Epoch 4/10
6129770/6129770 - 160s - loss: 0.1854 - binary_crossentropy: 0.1821 - val_loss: 0.1858 - val_binary_crossentropy: 0.1825
Epoch 5/10
6129770/6129770 - 158s - loss: 0.1853 - binary_crossentropy: 0.1819 - val_loss: 0.1858 - val_binary_crossentropy: 0.1824
Epoch 6/10
6129770/6129770 - 158s - loss: 0.1852 - binary_crossentropy: 0.1817 - val_loss: 0.1856 - val_binary_crossentropy: 0.1822
Epoch 7/10
6129770/6129770 - 143s - loss: 0.1851 - binary_crossentropy: 0.1815 - val_loss: 0.1856 - val_binary_crossentropy: 0.1821
Epoch 8/10
6129770/6129770 - 161s - loss: 0.1849 - binary_crossentropy: 0.1814 - val_loss: 0.1855 - val_binary_crossentropy: 0.1820
Epoch 9/10
6129770/6129770 - 158s - loss: 0.1848 - binary_crossentropy: 0.1812 - val_loss: 0.1855 - val_binary_crossentropy: 0.1820
Epoch 10/10
6129770/6129770 - 149s - loss: 0.1848 - binary_crossentropy: 0.1811 - val_loss: 0.1854 - val_binary_crossentropy: 0.1820
test LogLoss 0.1821
test AUC 0.6954
训练完毕...
deepFM训练耗时：1593.168582201004s

Process finished with exit code 0

Please refer to the [FAQ](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html) in doc and search for the [related issues](https://github.com/shenweichen/DeepCTR/issues) before you ask the question.

**Describe the question(问题描述)**
A clear and concise description of what the question is.

**Additional context**

**Operating environment(运行环境):**
 - python version [e.g. 3.6]
 - tensorflow version [e.g. 1.4.0, 1.15.0, 2.5.0]
 - deepctr version [e.g. 0.9.0,]

**Comment by zanshuxun** [comment]
有谁告诉你xDeepFM在任何数据上都一定比DeepFM效果好吗
